pkg/core/const.go

```diff
@@ -65,6 +65,7 @@ var (
 	ptrGitTrackedFlag             = flag.Bool("git-tracked", false, "Scan only git tracked files")
 	ptrPath                       = flag.String("path", utils.MustGetWD(), "Directory to scan (defaults to CWD) -- ABSOLUTE PATH ONLY")
 	ptrOutputFormat               = flag.String("format", "console", "Output format [ console | json | csv ]")
+	ptrWithConsole                = flag.Bool("withConsole", false, "While using --format, this flag will help to print findings in console")
```

Comment:
it looks like the flag name does not match the convention with others should this be `with-console`